meta_client/src/meta_impl.rs

```diff
@@ -132,10 +132,10 @@ impl MetaClient for MetaClientImpl {
         info!("Meta client finish creating table, resp:{:?}", pb_resp);
```

Comment:
finish --> finished

Reply:
Using `finished` here seems wrong in grammar.

Reply:
finished to create table?
I think such task has been done, past tense may be better.